src/lib/TranscriptEditor/MediaPlayer/PlaybackRate.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './PlaybackRate.module.css';
+class PlaybackRate extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+    render() {
+        return (
+          <div>
+            <p className={ styles.helpText }>Playback Rate
+              <b> <output >{ `x${ this.props.playBackRate }` }</output> </b>
+            </p>
+            <input
+                type="range"
+                min="0.2"
+                value={ this.props.playBackRate }
+                max="3.5"
+                step="0.1"
+                onChange={ this.props.handlePlayBackRateChange }
+                />
+            <br/>
```

Comment:
I know we have a bunch of <br/> tags in the HTML - keen to avoid em but they'll do until the css is in properly!

Reply:
Yeah, it's temporary until we figure out the look and feel and how to isolate the CSS scope